packages/appstore/src/components/main-title-bar/total-asset.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Text, Popover } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { isMobile } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import BalanceText from 'Components/elements/text/balance-text';
+import { useDelayedTotalAccountBalance, useCFDAccounts, usePlatformAccounts } from '@deriv/hooks';
+
+const TotalAsset = observer(() => {
+    const { traders_hub, client, common } = useStore();
+    const { selected_account_type, is_eu_user, no_CR_account, no_MF_account } = traders_hub;
+    const { default_currency } = client;
+    const { current_language } = common;
+    const { real: platform_real_accounts, demo: platform_demo_account } = usePlatformAccounts();
+    const { real: cfd_real_accounts, demo: cfd_demo_accounts } = useCFDAccounts();
+    const platform_real_balance = useDelayedTotalAccountBalance(platform_real_accounts, 60);
+    const cfd_real_balance = useDelayedTotalAccountBalance(cfd_real_accounts, 60);
+    const cfd_demo_balance = useDelayedTotalAccountBalance(cfd_demo_accounts, 60);
+    const is_real = selected_account_type === 'real';
+    const real_total_balance = platform_real_balance.balance + cfd_real_balance.balance;
```

Comment:
what if one of them is null? what is the default value here?

Reply:
the default value for each account balance is zero and if we have a situation which some required data like rates is not available yet, then we show loading instead of showing the wrong total_asset_amount to the client.  so we don't have to be worry about it